app/src/main/res/layout/selection_interaction_item.xml

```diff
@@ -19,24 +23,37 @@
     android:descendantFocusability="beforeDescendants"
     android:focusableInTouchMode="true"
     android:orientation="vertical"
-    android:padding="0dp"
+    android:paddingTop="12dp"
+    android:paddingBottom="12dp"
     app:layoutMarginEnd="@{viewModel.hasConversationView ? @dimen/margin_24 : @dimen/margin_32}"
     app:layoutMarginStart="@{viewModel.hasConversationView ? @dimen/margin_40 : @dimen/margin_32}"
     app:layout_constraintBottom_toBottomOf="parent"
     app:layout_constraintTop_toTopOf="parent">
 
+    <TextView
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      android:layout_marginStart="16dp"
```

Comment:
**Comment 6**
Whenever this item is shown we need extra `4dp` top/bottom margin because in mocks it shows `16dp` top/bottom of this textview.
For above `16dp`, `12dp` is already mentioned in **comment 5**  and `4dp` we are adding in this textview.

For below `16dp`, `12dp` will be covered as marginTop for first item of RecyclerView and `4dp` we are adding in this textview.